recipe/meta.yaml

```diff
@@ -29,15 +26,16 @@ requirements:
     - m2-automake{{ am_version }}  # [win]
     - m2-libtool  # [win]
     - m2w64-pkg-config  # [win]
-    - m2w64-toolchain  # [win]
     - pkg-config  # [not win]
     - posix  # [win]
-    - toolchain
+    - {{ compiler('c') }}        # [unix]
+    - {{ compiler('m2w64_c') }}  # [win]
```

Comment:
Ideally selectors would be respected here, but it seems they are not.

xref: https://github.com/conda/conda-build/issues/2958

Reply:
FWIW figuring out a workaround in PR ( https://github.com/conda-forge/backports.lzma-feedstock/pull/13 ).

Reply:
Thanks for the pointer! Windows is such a mess and I don't want to think about it :cry:

Reply:
This seems to do the trick. ( https://github.com/conda-forge/make-feedstock/commit/57d8e97123cf34cb72ad62d3dc23d400d7f762e7 )